QUICKSTART-ECS.md

```diff
@@ -0,0 +1,192 @@
+# Using a Bottlerocket AMI with Amazon ECS
+
+> The [ECS variant](./aws-ecs.md) of Bottlerocket is currently in a developer preview phase and we're looking for your
+> [feedback](https://github.com/bottlerocket-os/bottlerocket#contact-us).
+> We'd love for you to try it out!
+
+[Amazon Elastic Container Service (Amazon ECS)](https://ecs.aws) is a highly scalable, fast, container management service that makes it easy to run, stop, and manage Docker containers on a cluster.
+You can host your cluster on a serverless infrastructure that is managed by Amazon ECS by launching your services or tasks using the Fargate launch type.
+For more control over your infrastructure, you can host your tasks on a cluster of Amazon Elastic Compute Cloud (Amazon EC2) instances that you manage by using the EC2 launch type. Bottlerocket is available for the EC2 launch type.
```

Comment:
Should Fargate be mentioned here where folks reading how one uses Amazon ECS with *Bottlerocket* hosts? I'm worried this may cause some confusion. 

I'd be happy with just the pointers to the Amazon ECS docs taking up the torch on mentioning Fargate rather than adding them here. Is this out of consistency with the rest of the Amazon ECS docs?

Reply:
Yep, this text was mostly lifted from the intro [here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/Welcome.html).  I'm happy to change it to something else though.  (Or we could remove those sentences entirely?)

Reply:
This intro should stay scoped to the ECS bits that relate to Bottlerocket usages (at least in the QUICKSTART sense). The sprinkled pointers to the docs can help expand the subject and are better off introducing Fargate as there's more context to branch off that direction.

My vote is to remove the sentences and replace them with a shorter service introduction. Maybe something like:

```markdown
Amazon Elastic Container Service (Amazon ECS) is a highly scalable, fast, container management service that makes it easy to run, stop, and manage containers on a cluster.
Your containers are defined in a task definition which you use to run individual tasks or as a service.

This quickstart will walk through setting up an Amazon ECS cluster with Bottlerocket container instances (using EC2 Launch Type).
Check out the [Amazon ECS developer guide](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/Welcome.html) for more information on the supported Launch Types, additional details, and a broad overview of Amazon ECS.
```

<details>
<summary>rendered</summary>

Amazon Elastic Container Service (Amazon ECS) is a highly scalable, fast, container management service that makes it easy to run, stop, and manage containers on a cluster. Your containers are defined in a task definition which you use to run individual tasks or as a service.

This quickstart will walk through setting up an Amazon ECS cluster with Bottlerocket container instances (using EC2 Launch Type). Check out the [Amazon ECS developer guide](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/Welcome.html) for more information on the supported Launch Types, additional details, and a broad overview of Amazon ECS.

<details>

Reply:
Done, thanks for the suggestion!